src/cluster/src/helm.rs

```diff
@@ -101,6 +101,23 @@ impl HelmClient {
         })
     }
 
+    /// Get all the available versions
+    pub fn get_versions(&self, chart: &str) -> Result<Vec<Chart>, IoError> {
```

Comment:
name as chartVersions?

Reply:
I like either `chart_versions` or `find_versions`, or just `versions` works.

Reply:
versions it is